packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -294,10 +294,6 @@ class CupertinoTextField extends StatefulWidget {
   /// Whitespace characters (e.g. newline, space, tab) are included in the
   /// character count.
   ///
-  /// If [maxLengthEnforced] is set to false, then more than [maxLength]
```

Comment:
Why did this get removed?

Reply:
Because that applies to the Material text field, but not Cupertino.